base/math.jl

```diff
@@ -228,6 +228,30 @@ end
     return hi, lo
 end
 
+# generic, but involves double rounding
+function _180_over_pi(z::AbstractFloat)
+    180 / oftype(z, pi)
+end
+function _pi_over_180(z::AbstractFloat)
+    oftype(z, pi) / 180
+end
+
+# rounded to closest representable number where necessary
+function _180_over_pi(z::Union{Float16, Float32})
+    if z isa Float16
+        r = Float16(57.28)
+    elseif z isa Float32
+        r = 57.29578f0
+    end
+    r
+end
+function _pi_over_180(z::Union{Float16})
```

Comment:
Should just be `z::Float16` (and no if branch below, or is missing a `Float32`?

Reply:
Yeah, this was a weird style. Simplified per your suggestion.